### ISSUE TITLE
Typo / Solve ambiguities in  openssl/crypto/ec/curve448/scalar.c 

### ISSUE THREAD

**Issue by qualitesys** [openssl/openssl#13852]
The file openssl/crypto/ec/curve448/scalar.c  contains two subtile ambiguities.
In line 85:

```
chain += ((c448_dword_t) mand) * mier[j] + accum[j];
```

Is it possible to change to:
```
chain += ((c448_dword_t) mand) *mier[j] + accum[j];
```

In line 95:
```
chain += (c448_dword_t) mand *mier[j] + accum[j];
```
Is it possible to change to:
```
chain += ((c448_dword_t) mand) *mier[j] + accum[j];
```
Thanks for your help (my compiler gets trouble with it!)

**Comment by mspncp** [comment]
I'm not sure I understand which ambiguity you see in line 85 and what your suggested change should achieve: All I can see is a whitespace change, you removed a space between `*` and `mier[j]`. For the compiler, this change should make no difference. And for me, a space on both sides of the `*` makes more sense then your proposed change.

It is indeed a bit confusing at first sight that line 95 not only has different whitespace formatting, but also contains extra parentheses around the `(c448_dword_t) mand` cast. But those parentheses are redundant, because the cast operator has a higher precedence than the multiplication operator. So I don't see a reason why a compiler would complain about it. Could you please tell us which compiler you are using and post the exact text of the compiler warning?

**Comment by t8m** [comment]
Yeah, if removing the outer parentheses would help your compiler, that might be considered. However removing the space after `*` would be wrong and in conflict with the code formatting guide we have.

**Comment by t8m** [comment]
Also space after the cast could be removed.

**Comment by nhorman** [comment]
Marking as inactive, to be closed at the end of the 3.4 dev cycle barring further input

**Comment by t8m** [comment]
Closing as the issue was labeled as inactive for extended period of time and there was no additional input.
Please add a comment if you have any further information to support reopening the issue.
